db/migrate/20230117043026_create_comments.rb

```diff
@@ -0,0 +1,10 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text
+      t.references :post, null: false, foreign_key: true
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] Same here, the associations between models must be set up correctly according to the given [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png).  To show that, the  `'author_id'` reference the User class . Kindly write the query like this so the foreign key name will be `author_id` :
```ruby
   add_reference :comments, :author, null: false, foreign_key: { to_table: :users }
```
- This will create author_id and a corresponding index